spec.bs

```diff
@@ -231,6 +233,27 @@ previous position or size of any previously closed |pip traversable| window.
     and is <code>true</code>, the user agent should not display UI affordances
     on the picture-in-picture window that allow the user to return to the
     opener window.
+15. If |options|["{{DocumentPictureInPictureOptions/createHidden}}"] exists
+    and is <code>true</code>, the user agent should not show the picture-in-picture
+    window immediately.  Instead, it will wait until one of these becomes true:
+    1. {{DocumentPictureInPicture/show()}} is called, with the corresponding
+        window as its argument.
+    2. The transient activation used in step 4 would time out.
```

Comment:
If you want to force all UAs to always time out in that way this can be made more specific:

Before we consume transient activation in step 7, store `this`'s `relevant global object`'s `last activation timestamp` in a variable. Then here say once the `current high resolution time` given `this`'s `relevant global object` becomes greater than or equal to that variable plus the `transient activation duration`.

Alternatively, you can make this less specific and say that the UA may decide to time out and just show the window.

Reply:
updated to go with "UA chooses", since it's all somewhat of a race anyway.